lib/p2p/Peer.ts

```diff
@@ -267,23 +276,23 @@ class Peer extends EventEmitter {
   /**
    * Wait for a packet to be received from peer.
    */
-  private addResponseTimeout = (packetType: PacketType, timeout: number): PendingResponseEntry | null => {
+  private addResponseTimeout = (key: string, timeout: number): PendingResponseEntry | null => {
```

Comment:
If the key passed to this method is always expected to be a hash, I think I'd prefer naming it `hash` within this method rather than `key`.

Reply:
Fixed.